pkg/controller/backup_sync_controller.go

```diff
@@ -272,10 +273,20 @@ func (b *backupSyncReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 			for _, snapCont := range snapConts {
 				// TODO: Reset ResourceVersion prior to persisting VolumeSnapshotContents
 				snapCont.ResourceVersion = ""
-				// For creating static VSContent, we should put snapshothandle in source rather than volume handle.
-				// Because if VSContent syncs to a different cluster, having volumeHandle will force rePUTs on the snapshot
-				snapCont.Spec.Source.SnapshotHandle = snapCont.Status.SnapshotHandle
-				snapCont.Spec.Source.VolumeHandle = nil
+				// Make the VolumeSnapshotContent static
+				snapCont.Spec.Source = snapshotv1api.VolumeSnapshotContentSource{
+					SnapshotHandle: snapCont.Status.SnapshotHandle,
+				}
+				// Set VolumeSnapshotRef to none exist one, because VolumeSnapshotContent
+				// validation webhook will check whether name and namespace are nil.
+				// external-snapshotter needs Source pointing to snapshot and VolumeSnapshot
+				// reference's UID to nil to determine the VolumeSnapshotContent is deletable.
+				snapCont.Spec.VolumeSnapshotRef = corev1api.ObjectReference{
+					APIVersion: snapshotv1api.SchemeGroupVersion.String(),
+					Kind:       "VolumeSnapshot",
+					Namespace:  "ns-" + string(snapCont.UID),
+					Name:       "name-" + string(snapCont.UID),
```

Comment:
Is this snapshot namespacedname supposed to be non-existing?
Can the string here be anything?

Reply:
@kaovilai this is a copy paste from -https://github.com/vmware-tanzu/velero/blame/824bebbad7bc61077c6e80731468fb020d3dfd4f/pkg/util/csi/volume_snapshot.go#L641

My understanidng is that any name should work here. 
We are creating a static VSC